docs/decisions/0009-usage-of-react-bootstrap.rst

```diff
@@ -0,0 +1,73 @@
+5. Usage of React-Bootstrap
```

Comment:
I am overall supportive of this decision.
If this ADR is accepted, as part of this PR, please update the `Status` of ADR 0008 to `Superseded by ADR-0009`.

Reply:
Will do.